docs/source/coredev/index.rst

```diff
@@ -22,7 +22,21 @@ Backporting Pull requests
 
 All pull requests should usually be made against ``master``, if a Pull Request
 need to be backported to an earlier release; then it should be tagged with the
-correct ``milestone``. We then use `ghpro <https://pypi.python.org/pypi/ghpro>`
+correct ``milestone``.
+
+If you are an admin on the IPython repository just mention the backport bot to
+do the work for you. The bot is evolving so instruction may be different. At
+the time of this writing you can use::
+
+    @meeseeksdev[bot] backport to <branchname>
```

Comment:
yes it works without the `[bot]` ot the `@`. GitHub just happend to autocomplete with both when the bot is part of a conversation.